railsApi.js

```diff
@@ -1,14 +1,15 @@
-function inizializeRequest(url, data, typeget, successFunction, errorFunction) {
+function inizializeRequest(url, data,dataType, typeget, headers, successFunction) {
   $.ajax({
     url: `http://binar-taste-api-staging.builder01.binarapps.com/api/v1${url}`,
     data: data,
     type: typeget,
     crossDomain: true,
-    dataType: 'json',
     async: false,
     headers: {
       'Cache-Control': 'max-age=1000',
+      ...headers,
```

Comment:
spread operator (`...`) is not working in ES5 that we're working on (as long as you don't have Babel in project, you work in ES5).  here you have some es5 equivalents for methods implemented in es6 https://github.com/addyosmani/es6-equivalents-in-es5#spread-operator